shim-array.js

```diff
@@ -131,8 +142,77 @@ define("deleteAll", function (value, equals) {
     }
     return count;
 });
+// https://tc39.github.io/ecma262/#sec-array.prototype.find
+if (!Array.prototype.find) {
```

Comment:
Indentation style is not consistent below. Consider `unexpand -t 2 | expand -t 4`.

Reply:
fix (THX)